components/bl-calendarHeatmap-component/src/index.js

```diff
@@ -0,0 +1,91 @@
+import { useMemo, useEffect, useRef, useState, useCallback } from 'react';
+import CalendarHeatmap from './lib/react-calendar-heatmap.umd.min';
+import ReactTooltip from './lib/react-tooltip.min';
+import { Legend } from './subcomponents';
+import { validate, shadeColor, shiftDate } from './helpers';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const today = new Date();
+
+export default function CalendarHeatmapComponent({ component, eventHandlers }) {
+  const {
+    style,
+    display,
+    classList,
+    calendarData,
+    monthLabels,
+    weekdayLabels,
+    color,
+    legend,
+    showMonthLabels,
+    showWeekdayLabels
+  } = component;
```

Comment:
```suggestion
  const {
    style,    display,    classList,    calendarData,    monthLabels,
    weekdayLabels,    color,    legend,    showMonthLabels,    showWeekdayLabels
  } = component;
```

What you think about it?